src/test/java/io/axoniq/demo/gamerental/controller/GameRentalRestControllerTest.java

```diff
@@ -34,7 +38,18 @@ void setUp() {
         commandGateway = mock(ReactorCommandGateway.class);
         queryGateway = mock(ReactorQueryGateway.class);
 
-        testClient = WebTestClient.bindToController(new GameRentalRestController(commandGateway, queryGateway)).build();
+        testClient = WebTestClient.bindToController(new GameRentalRestController(commandGateway, queryGateway))
+                .httpMessageCodecs(this::jsonCodecs)
+                .configureClient().codecs(this::jsonCodecs)
+                .build();
```

Comment:
it also might pick up that module if you build `WebTestClient` from spring's `@AutoconfigureWebTestClient` but it's probably slower